pkgs/top-level/perl-packages.nix

```diff
@@ -19420,6 +19420,12 @@ let
       url = "mirror://cpan/authors/id/M/MA/MARSCHAP/perl-ldap-0.66.tar.gz";
       hash = "sha256-CSY85hZugMmNaJ1B0JmVuBM4n9Bpt4RgH23Ff44rQQI=";
     };
+    # ldapi socket location should match the one compiled into the openldap package
+    postPatch = ''
+      for f in lib/Net/LDAPI.pm lib/Net/LDAP/Util.pm lib/Net/LDAP.pod lib/Net/LDAP.pm; do
+        sed -i 's:/var/run/ldapi:/run/openldap/ldapi:g' "$f"
```

Comment:
```suggestion
        sed -i 's:/run/ldapi:/run/openldap/ldapi:g' "$f"
```

Reply:
Hm? That would result in `/var/run/openldap/ldapi`. We could just not use the compat symlink anymore

Reply:
Yeah, that was the intent.